src/main/java/edu/usf/cutr/gtfs_realtime/bullrunner/GtfsRealtimeProviderImpl.java

```diff
@@ -83,210 +80,305 @@
+                Position.Builder position = Position.newBuilder();
+                VehicleDescriptor.Builder VehicleInfo = VehicleDescriptor.newBuilder();
+                FeedEntity.Builder vehiclePositionEntity = FeedEntity.newBuilder();
+
+                // set values for feed
+                tripDescriptor.setRouteId(route_id);
+                position.setBearing((float) vehicleObj.getDouble("headingDegrees"));
+                position.setLatitude((float) vehicleObj.getDouble("lat"));
+                position.setLongitude((float) vehicleObj.getDouble("lon"));
+                position.setSpeed((float) vehicleObj.getDouble("speed"));
+                VehicleInfo.setId(String.valueOf(vehicleObj.getInt("id")));
+                VehicleInfo.setLabel(vehicleObj.getString("name"));
+
+                // Build feed
+                VehiclePosition.Builder VehiclePosition_route = VehiclePosition.newBuilder();
+                VehiclePosition_route.setPosition(position);
+                VehiclePosition_route.setTrip(tripDescriptor);
+                VehiclePosition_route.setVehicle(VehicleInfo);
+                VehiclePosition_route.setTimestamp(
+                        Instant.parse(vehicleObj.getString("lastUpdated")).getEpochSecond()
+                );
+                if (vehicleObj.getDouble("passengerLoad") <= 0)
+                    VehiclePosition_route.setOccupancyStatus(OccupancyStatus.EMPTY);
+                else if (vehicleObj.getDouble("passengerLoad") <= 50)
+                    VehiclePosition_route.setOccupancyStatus(OccupancyStatus.MANY_SEATS_AVAILABLE);
+                else if (vehicleObj.getDouble("passengerLoad") <= 70)
+                    VehiclePosition_route.setOccupancyStatus(OccupancyStatus.FEW_SEATS_AVAILABLE);
+                else if (vehicleObj.getDouble("passengerLoad") <= 90)
+                    VehiclePosition_route.setOccupancyStatus(OccupancyStatus.STANDING_ROOM_ONLY);
+                else if (vehicleObj.getDouble("passengerLoad") <= 95)
+                    VehiclePosition_route.setOccupancyStatus(OccupancyStatus.CRUSHED_STANDING_ROOM_ONLY);
+                else VehiclePosition_route.setOccupancyStatus(OccupancyStatus.FULL);
+
+                vehicleFeedID++;
+                vehiclePositionEntity.setId(Integer.toString(vehicleFeedID));
+                vehiclePositionEntity.setVehicle(VehiclePosition_route);
+                vehiclePositions.addEntity(vehiclePositionEntity.build());
+            }
         }
+        _vehiclePositionsSink.handleFullUpdate(vehiclePositions);
+        _log.info("vehicles' location extracted: " + vehiclePositions.getEntities().size());
+        // ---- END UPDATING VEHICLE POSITIONS -----------------
+
+        // ----- START UPDATING TRIP UPDATES, LEGACY FROM VERSION 0.9.0 ------------
+        // set to null for version 1.0.0
+        GtfsRealtimeFullUpdate tripUpdates = new GtfsRealtimeFullUpdate();
+        /*

```

Comment:
Could you please delete all the commented-out code and unused methods for TripUpdates too?  If we want to go back and see the implementation we can always pull it up via Git history.